packages/storybook/src/migrations/update-12-1-0/fix-storybook-tsconfig.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { Tree } from '@angular-devkit/schematics';
+import { getFileContent } from '@nrwl/workspace/testing';
+import { runMigration } from '../../utils/testing';
+
+describe('Fix Storybook TSConfig to avoid VSCode error', () => {
+  let tree: Tree;
+
+  beforeEach(async () => {
+    tree = Tree.empty();
+
+    // create workspace
```

Comment:
I think, we should avoid useless comments. The code should be self explanatory, if it isn't the case, we better refactor it in functions/variables.

In this case, I think, we can just remove this comment, the code is already clear enough, we don't need to refactor more.